.cirrus.yml

```diff
@@ -265,6 +265,26 @@ task:
       script:
         - ./dev/bots/firebase_testlab.sh
 
+    - name: web_smoke_test
+      only_if: "changesInclude('.cirrus.yml', 'examples/hello_world/**' ,'dev/**', 'packages/flutter/**', 'packages/flutter_test/**', 'packages/flutter_tools/lib/src/test/**', 'packages/flutter_web_plugins/**', 'bin/internal/**') || $CIRRUS_PR == ''"
+      environment:
+        # Tests Hello World with Flutter Web Engine using Flutter Driver. Should not need more resources.
+        CPU: 2
+        MEMORY: 2G
+        CHROME_NO_SANDBOX: true
+      script:
+        - git clone https://github.com/flutter/web_installers.git
+        - cd web_installers/packages/web_drivers/
+        - pub get
+        - dart lib/web_driver_installer.dart &
+        - sleep 20
+        - chromedriver/chromedriver --port=4444 &
+        - sleep 5
+        - cd ../../../examples/hello_world/
+        - flutter run -v -d web-server --profile --local-engine=host_debug_unopt --target=test_driver/smoke_web_engine.dart --web-port=8080 &
```

Comment:
I think you'll want to `export FLUTTER_WEB=true` before running any of the web commands. Otherwise the tool will refuse to do any web stuff.

Reply:
the run is temporary. so this will also be gone. we won't need it one the PR is in: https://github.com/flutter/flutter/pull/51084 (I won't merge my PR before that one)

Reply:
Now PR #51084 is in, no need for flutter run!